src/style.css

```diff
@@ -0,0 +1,101 @@
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Book.ttf");
+  font-weight: 300;
+}
+@font-face {
+  font-family: "Avenir";
+  src: url("fonts/Avenir-Heavy.ttf");
+  font-weight: bold;
+}
+
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+  font-weight: 300;
+}
+
+.searchbar {
+  display: flex;
+  height: 70px;
+  align-items: center;
+  border-radius: 4px;
+  border: 1px solid #e1e7ed;
+  margin: 20px 8px;
+  box-shadow: 0 1px 8px 0 rgba(61, 78, 97, 0.1);
+  overflow: hidden;
+  &:hover {
+    box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.25);
+  }
+
+  &:focus-within {
+    border: none;
+    background: linear-gradient(180deg, #fff 0%, #f6f6f7 100%);
+  }
+
+}
+
+.searchbar__label {
+  display: flex;
+  flex-shrink: 0;
+  padding: 9.5px;
+  margin-left: 25px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-position: center;
+  background-size: contain;
+}
+
+.searchbar__input {
+  width: 100%;
+  border: none;
+  font-family: "Avenir", sans-serif;
+  font-size: 16px;
+  padding-left: 18px;
+
+  &:focus {
+    text-shadow: 0 4px 4px rgba(0, 0, 0, 0.25);
+    color: #3d4e61;
+    font-family: "Avenir", sans-serif;
+    font-size: 1rem;
```

Comment:
Decide if you want to use `rem` for font-size or `px`. Usually it's `rem` for font-sizes